variables.tf

```
@@ -0,0 +1,86 @@
+variable "es_endpoint" {}
+variable "es_domain_arn" {}
+variable "es_security_group_id" {}
+
+variable "schedule" {
+  default = "cron(0 3 * * ? *)"
+}
+
+variable "vpc_id" {
+  description = "VPC Id for the Lambda function - should be the same VPC as ES domain"
+}
+
+variable "sns_alert" {
+  description = "SNS ARN to pusblish any alert"
+  default     = ""
+}
+
+variable "index" {
+  description = "Index/indices to process comma separated, with all every index will be processed except '.kibana'"
+  default     = "all"
+}
+
+variable "delete_after" {
+  description = "Numbers of days to preserve"
+  default     = 15
+}
+
+variable "enabled" {
+  type        = "string"
+  description = "Enable or disable the ElasticSearch Cleanup function"
```

```suggestion
  description = "Enable or disable the Elasticsearch Cleanup function"
```

Seems like the canonical spelling is `Elasticsearch` per AWS documentation